tests/shell/tests.py

```diff
@@ -58,6 +69,18 @@ def test_stdin_read_globals(self, select):
             call_command("shell")
         self.assertEqual(stdout.getvalue().strip(), "True")
 
+    @unittest.skipIf(
+        sys.platform == "win32",
+        "Windows select() doesn't support file descriptors.",
+    )
+    @mock.patch("django.core.management.commands.shell.select")  # [1]
+    def test_stdin_read_globals_import(self, select):
+        with captured_stdin() as stdin, captured_stdout() as stdout:
+            stdin.write(self.script_globals_import)
```

Comment:
This test is redundant, as is `self.script_globals_import`. `test_stdin_read_imports` covers the functionality already, and is IMO clearer. Using `Phone` is the same as checking it is available in globals.